codec-haproxy/src/main/java/io/netty/handler/codec/haproxy/HAProxyMessage.java

```diff
@@ -519,4 +516,61 @@ public int destinationPort() {
     public List<HAProxyTLV> tlvs() {
         return tlvs;
     }
+
+    @Override
+    public HAProxyMessage touch() {
+        tryRecord();
+        return (HAProxyMessage) super.touch();
+    }
+
+    @Override
+    public HAProxyMessage touch(Object hint) {
+        tryRecord();
```

Comment:
for this you need to duplicate the code as you should pass down the `hint`.

Reply:
I am sorry for my carelessness. Done!